PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/toolkit/intellij/function/action/CreateFunctionAppAction.java

```diff
@@ -92,13 +92,20 @@ private void openDialog(final Project project, @Nullable final FunctionAppConfig
     @AzureOperation(name = "function.create_detail", params = {"config.getName()"}, type = AzureOperation.Type.ACTION)
     private Single<FunctionApp> createFunctionApp(final FunctionAppConfig config) {
         final IAzureOperationTitle title = title("function.create_detail", config.getName());
+        final IntellijAzureMessager actionMessenger = new IntellijAzureMessager() {
+            @Override
+            public void info(@NotNull String message, String title) {
+                // swallow info notification for action
+            }
+        };
         final AzureTask<FunctionApp> task = new AzureTask<>(null, title, false, () -> {
+            AzureMessager.getContext().setMessager(actionMessenger);
             final ProgressIndicator indicator = ProgressManager.getInstance().getProgressIndicator();
             indicator.setIndeterminate(true);
             return functionAppService.createFunctionApp(config);
         });
         return AzureTaskManager.getInstance().runInModalAsObservable(task).toSingle().doOnSuccess(app -> {
-            this.notifyCreationSuccess(app);
+            AzureMessager.getMessager().success(message("function.create.success.title"), message("function.create.success.message", app.name()));
```

Comment:
be careful about the order of the parameters of `IAzureMessager#success(message, title)`